mods/d2k/rules/structures.yaml

```diff
@@ -599,7 +598,7 @@ starport:
 	Valued:
 		Cost: 1500
 	Building:
-		Footprint: xxx x=x =x=
+		Footprint: xxx xxx =x=
```

Comment:
It seems better not to move units into middle of startport, but the commit comment 'allowed both friendly and enemy actors to block the exit' seems incorrect - in current release unit in middle doesn't prevent leaving new units according to my test.